CMakeLists.txt

```diff
@@ -78,6 +78,10 @@ option(WITH_HIGH_LEVEL_API_TEST   "Test fluid python high-level api interface"
 option(PY_VERSION       "Compile PaddlePaddle with python3 support"     ${PY_VERSION})
 option(WITH_FAST_MATH   "Make use of fast math library, might affect the precision to some extent" ON)
 option(WITH_DGC   "Use DGC(Deep Gradient Compression) or not" ON)
+option(SANITIZE_ADDRESS "Enable AddressSanitizer for sanitized targets" OFF)
+option(SANITIZE_LEAK "Enable LeakSanitizer for sanitized targets" OFF)
+option(SANITIZE_MEMORY "Enable MemorySanitizer for sanitized targets" OFF)
+option(SANITIZE_THREAD "Enable ThreadSanitizer for sanitized targets." OFF)
```

Comment:
Could you remove the option, and write like `CMAKE_BUILD_TYPE`
https://github.com/PaddlePaddle/Paddle/blob/047bba855b1cdd99142ec7b15a4f19015378c5b8/CMakeLists.txt#L83-L87

Reply:
You don't need `Sanitizers.cmake` file to include Sanitizers support in PaddlePaddle, it is built in gcc compiler.
I moved many options to one option SANITIZER_TYPE.

Reply:
> it is built in gcc compiler.

Does gcc48 support it?

Reply:
Yes, it's a part of GCC starting from version 4.8